release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -188,6 +188,14 @@ As a cluster administrator, you can enable cluster capabilities to select or des
 
 For more information, see xref:../post_installation_configuration/cluster-capabilities.adoc#cluster-capabilities[Cluster capabilities].
 
+[id="ocp-4-11-heterogeneous-tech-preview"]
+==== {product-title} on heterogeneous architectures (Technology Preview)
+{product-title} 4.11 introduces heterogeneous architecture cluster support using Azure installer-provisioned infrastructures in Technology preview. This feature offers, as a day two operation, the ability to add `arm64` worker nodes to an existing `x86_64` Azure cluster that is installer provisioned with a heterogeneous installer binary. You can add `arm64` workers to your heterogeneous cluster by creating a custom Azure machine set that uses a manually generated `arm64` bootimage. Control planes on `arm64` architectures are not currently supported. For more information, see xref:../post_installation_configuration/deploy-heterogeneous-configuration.adoc[Configuring a heterogeneous cluster].
```

Comment:
```suggestion
{product-title} 4.11 introduces heterogeneous architecture cluster support using Azure installer-provisioned infrastructure in Technology preview. This feature offers, as a day two operation, the ability to add `arm64` worker nodes to an existing `x86_64` Azure cluster that is installer provisioned with a heterogeneous installer binary. You can add `arm64` workers to your heterogeneous cluster by creating a custom Azure machine set that uses a manually generated `arm64` bootimage. Control planes on `arm64` architectures are not currently supported. For more information, see xref:../post_installation_configuration/deploy-heterogeneous-configuration.adoc[Configuring a heterogeneous cluster].
```

Reply:
@kelbrown20 take a look at this for the previous suggestion: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#installer-provisioned-infrastructure